components/core/src/os/filesystem/windows.rs

```diff
@@ -17,7 +17,7 @@ use std::ffi::CStr;
 use std::path::Path;
 use std::io;
 
-pub fn chown(r_path: *const c_char, uid: u32, gid: u32) -> c_int {
+pub fn chown(r_path: *const c_char, uid: String, gid: String) -> c_int {
```

Comment:
@smurawski is right, the interface for a function which is not marked unsafe should not include raw pointers. I like your approach just fine, though, of just getting things in and moving and coming around and fixing them. Please leave a `MW TODO: <msg>` comment wherever you intend to do that so we can easily find them 😄